rules/aws/persistence_ec2_security_group_configuration_change_detection.toml

```diff
@@ -0,0 +1,46 @@
+[metadata]
+creation_date = "2021/05/05"
+maturity = "development"
+updated_date = "2021/05/05"
+
+[rule]
+author = ["Elastic", "Austin Songer"]
+description = """
+Detects threat actors making configuration change to EC2 security groups. Threat actors may be able to maintain
+persistence, exfiltrate, and pivot in a AWS environment.
+"""
+false_positives = ["Unknown"]
+from = "now-10m"
+index = ["filebeat-*", "logs-aws*"]
+interval = "10m"
+language = "kuery"
+license = "Elastic License v2"
+name = "AWS EC2 Security Group Configuration Change Detection"
+note = "The AWS Filebeat module must be enabled to use this rule."
+references = ["https://docs.aws.amazon.com/ec2/index.html"]
+risk_score = 21
+rule_id = "29052c19-ff3e-42fd-8363-7be14d7c5469"
+severity = "low"
+tags = ["Elastic", "Cloud", "AWS", "Continuous Monitoring", "SecOps", "Network Security"]
+timestamp_override = "event.ingested"
+type = "query"
+
+query = '''
+event.dataset:aws.cloudtrail and event.provider:iam.amazonaws.com and event.action:(AuthorizeSecurityGroupIngress or AuthorizeSecurityGroupEgress or CreateSecurityGroup or ModifyInstanceAttribute) and event.outcome:success
```

Comment:
This will likely be noisy, especially the `AuthorizeSecurityGroupIngress` API call.

Reply:
It falls in line with the rule, because `AuthorizeSecurityGroupIngress` call only happens when someone adds the specified ingress rules to a security group, so that is something that would not being happening everyday.